### DISCUSSION TITLE
Git hub page URL is not working after deleted repos and creating new one with same name

### DISCUSSION THREAD

**Discussion by abmptit** [community/community#30373] [Other Feature Feedback, Questions, & Ideas]
Hello,
I have an issue with github pages, here the steps:
1/ I created repos TOTO, I deployed github page https://USER.github.io/toto
2/ After that I removed the TOTO repos, I recreate another with same name, I deployed github pages,
3/ When I try to navigate to  https://USER.github.io/toto I'm redirected to https://laughing-carnival-953f0f88.pages.github.io/
Any idea plz ?
Thanks

**Comment by yoannchaudet** [comment]
The URL you got redirected to is a Private Pages URL. This could be a bug, I cannot tell without knowing the repository name.

Would you mind opening a [support ticket](https://support.github.com/features/pages) (contact us button at the bottom) so we can help you?
